engine-mr/src/main/java/org/apache/kylin/engine/mr/streaming/SaveDictStep.java

```diff
@@ -136,6 +136,7 @@ public boolean accept(Path path) {
                         logger.error("dictionary of column {} not found! ", colRef.getName());
                     }
                 }
+                reader.close();
```

Comment:
Will it be more safe to put it into a "finally" block?

Reply:
Thank you!